### ISSUE TITLE
Leather Horse Armor not registering as acceptable itemtype

### ISSUE THREAD

**Issue by TheCashFire** [GC-spigot/AdvancedEnchantments#4292]
### Describe the bug

Expecting to be able to use LEATHER_HORSE_ARMOR as an item for the config.yml "armor-items" section. I can get basically anything else to apply, but leather horse armor won't. I use it as a base for custom hats, so if this could be fixed, that would be great.

### How to reproduce

Add it to the armor-items: section.

armor-items:
  helmet:
  - HEAD
  - SKULL_ITEM
  - SKULL
  - PLAYER_HEAD
  - CREEPER_HEAD
  - DRAGON_HEAD
  - ZOMBIE_HEAD
  - NETHERITE_HELMET
  - DIAMOND_HELMET
  - GOLD_HELMET
  - GOLDEN_HELMET
  - IRON_HELMET
  - CHAINMAIL_HELMET
  - LEATHER_HELMET
  - TURTLE_SHELL
  - PUMPKIN
  - CARVED_PUMPKIN
  - TURTLE_HELMET
  - LEATHER_HORSE_ARMOR

### "/ae plinfo" link

https://paste.md-5.net/sawazihuce

**Comment by ThomasWega** [comment]
What enchantment are you trying to put on it? What have you tested?

**Comment by TheCashFire** [comment]
Anything. Head based enchantments (for instance, Aquatic), Armor based enchantments (for instance, Reflect), a handful of enchantments of my own making, no matter where LEATHER_HORSE_ARMOR gets put in the config, it is entirely unenchantable because armor-items does not recognize it for some reason. Won't work for drag and drop, won't work for anvils, I can't even use a command to force an enchantment onto it because its not classified as a legal item to enchant (This enchantment cannot be applied tot his item.) even though I have LEATHER_HORSE_ARMOR listed in the config, in the enchantment config itself, everywhere.

What else have I tried? Lets see.
Instead of LEATHER_HORSE_ARMOR, I tried STONE. Worked fine, could enchant stone after I added it into the armor-items section I wanted. Piece of cake.
Tried KELP, an item. Works fine.
Tried SPONGE, a block. Works fine.
Tried all other variants of horse armor, Iron, Gold, Diamond. Doesn't work on any of those either.
